### ISSUE TITLE
Names with Spaces

### ISSUE THREAD

**Issue by Haxley** [poggit-orphanage/BuycraftPM#1]
Anyone know how to let players with spaces in their names purchase from the buycraft store? Every time they type their name in it says that it’s not in the right format

**Comment by Awzaw** [comment]
Did you try with quotes?! I've contacted them and will post the reply here

**Comment by Haxley** [comment]
Yeah quotes doesn't work but now that I think about it, it probably isn't a problem with anything here. The problem is with the web interface. It won't let you type your name on the web store if it has spaces.

**Comment by Awzaw** [comment]
I suspect the plugin will have to be updated too

**Comment by Haxley** [comment]
From Buycraft support: “Thanks for the email about this, we are currently building support for this and completely revamping all of our name handling systems which will be released early January.”
